Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -1114,6 +1114,38 @@ public KeltnerChannels KCH(Symbol symbol, int period, decimal k, MovingAverageTy
             return keltnerChannels;
         }
 
+        /// <summary>
+        /// Creates a new KnowSureThing indicator for the symbol. The indicator will be automatically
+        /// updated on the given resolution.
+        /// </summary>
+        /// <param name="symbol">The symbol whose KST we want</param>
+        /// <param name="roc1Period">The period over which to compute ROC1</param>
+        /// <param name="roc1MaPeriod">The smoothing period used to smooth the computed ROC1 values</param>
+        /// <param name="roc2Period">The period over which to compute ROC2</param>
+        /// <param name="roc2MaPeriod">The smoothing period used to smooth the computed ROC2 values</param>
+        /// <param name="roc3Period">The period over which to compute ROC3</param>
+        /// <param name="roc3MaPeriod">The smoothing period used to smooth the computed ROC3 values</param>
+        /// <param name="roc4Period">The period over which to compute ROC4</param>
+        /// <param name="roc4MaPeriod">The smoothing period used to smooth the computed ROC4 values</param>
+        /// <param name="signalPeriod">The smoothing period used to smooth the signal values</param>
+        /// <param name="movingAverageType">Specifies the type of moving average to be used as smoother for KnowSureThing values</param>
+        /// <param name="resolution">The resolution</param>
+        /// <param name="selector">Selects a value from the BaseData to send into the indicator, if null defaults to casting the input value to a TradeBar</param>
+        /// <returns>A new KnowSureThing indicator with the specified smoothing type and period</returns>
+        [DocumentationAttribute(Indicators)]
+        public KnowSureThing KST(Symbol symbol,
+            int roc1Period, int roc1MaPeriod, int roc2Period, int roc2MaPeriod,
+            int roc3Period, int roc3MaPeriod, int roc4Period, int roc4MaPeriod, int signalPeriod,
```

Comment:
Think we could set some defaults values for the roc and ma periods? https://www.investopedia.com/terms/k/know-sure-thing-kst.asp